test/test_gguf_format.py

```diff
@@ -192,6 +192,49 @@ def test_q6_k(self):
         inputs = self.tokenizer(text, return_tensors="pt").to(model.device)
         print(self.tokenizer.decode(model.generate(**inputs, max_new_tokens=10)[0]))
         shutil.rmtree("./saved", ignore_errors=True)
+    
+    def test_gguf_baseline(self):
+        model_name = "Qwen/Qwen2.5-1.5B-Instruct" 
+        model_name = "/models/Qwen2.5-1.5B-Instruct" 
+        model = AutoModelForCausalLM.from_pretrained(model_name, torch_dtype="auto", trust_remote_code=True)
+        autoround = AutoRound(
+            model,
+            self.tokenizer,
+            bits=3,
+            group_size=16,
+            sym=True,
+            iters=0,
+            data_type="gguf_int_sym_dq",
+            super_group_size=16,
+            super_bits=6
+        )
+        quantized_model_path = "./saved"
+        autoround.quantize_and_save(output_dir=quantized_model_path, inplace=False, format="fake")
+        model = AutoModelForCausalLM.from_pretrained(quantized_model_path, device_map="auto")
+        text = "There is a girl who likes adventure,"
+        inputs = self.tokenizer(text, return_tensors="pt").to(model.device)
+        print(self.tokenizer.decode(model.generate(**inputs, max_new_tokens=10)[0]))
+        shutil.rmtree("./saved", ignore_errors=True)
+    
+        autoround = AutoRound(
+            model,
+            self.tokenizer,
+            bits=5,
+            group_size=32,
+            sym=True,
+            iters=0,
+            data_type="int_asym_dq",
+            super_group_size=8,
+            super_bits=6
+        )
+        quantized_model_path = "./saved"
+        autoround.quantize_and_save(output_dir=quantized_model_path, inplace=False, format="gguf:q5_k_s,fake")
```

Comment:
why export two formats?
better add a cuda ut later that could show the accuracy improvement